### PR TITLE
fix: add an empty state to the project list

### PR DESCRIPTION
**Pull request by hashrock**
This PR adds empty states to the project list, it guides new users after a fresh install.

image
image

### CODE REVIEW

File: routes/index.tsx [hunk 1]
```diff
@@ -88,6 +89,24 @@ export default function HomePage(props: PageProps<HomePageData>) {
       <Head href={props.url.href} />
       <main class="flex-1 p-4">
         <TimeSelector url={props.url} />
+        {props.data.items.length === 0 && (
+          <>
+            <div class="flex flex-col justify-center items-center gap-2">
+              <div class="flex justify-center gap-2 pt-16">
+                <Info class="w-6 h-6" />
+                <p class="text-center font-medium">No items found.</p>
```
Comment (iuioiua) [inline routes/index.tsx:97]:
```suggestion
                <p class="text-center font-medium">No items found</p>
```
Nit

File: routes/index.tsx [hunk 2]
```diff
@@ -88,6 +89,24 @@ export default function HomePage(props: PageProps<HomePageData>) {
       <Head href={props.url.href} />
       <main class="flex-1 p-4">
         <TimeSelector url={props.url} />
+        {props.data.items.length === 0 && (
+          <>
+            <div class="flex flex-col justify-center items-center gap-2">
+              <div class="flex justify-center gap-2 pt-16">
+                <Info class="w-6 h-6" />
```
Comment (iuioiua) [inline routes/index.tsx:96]: Perhaps, the icon would look nicer placed on the line above and centered. WDYT? I don't feel strongly.

**Comment by hashrock** [comment]
Updated styles, wdyt?

image
image

File: routes/index.tsx (hunk 2 above)
Comment (hashrock) [reply routes/index.tsx]: Yeah! agreed.

**Review by iuioiua — APPROVED** [review]
Perfect! Thank you 🙏🏾

**Pull request merged**